app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -3522,30 +3522,6 @@ class BrowserTabViewModelTest {
         }
     }
 
-    @Test
-    fun whenConsumeAliasThenPixelSent() {
```

Comment:
Tested elsewhere, in https://github.com/duckduckgo/Android/pull/3582/files#diff-86ed09c4f784467e8f157d920ab5dbfcedbf2995ff42181622cf763407052d50